### PR TITLE
Removed calls to lowercase media URLs

### PR DESCRIPTION
**Pull request by clintonb**
These URLs are case-sensitive and should not be modified.

ECOM-4266

### CODE REVIEW

**Comment by clintonb** [comment]
FYI @bderusha @peter-fogg @PaulWattenberger

**Comment by clintonb** [comment]
@bderusha @peter-fogg please review the orphan deletion commit

File: course_discovery/apps/course_metadata/data_loaders.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def update_organization(self, body):
         image = None
         image_url = body['logo']
         if image_url:
-            image_url = image_url.lower()
+            image_url = image_url
```
Comment (peter-fogg) [inline course_discovery/apps/course_metadata/data_loaders.py:123]: I guess I missed this on Friday, but why leave this noop line here?

Comment (clintonb) [inline course_discovery/apps/course_metadata/data_loaders.py:123]: (facepalm)

Comment (clintonb) [inline course_discovery/apps/course_metadata/data_loaders.py:123]: Moving fast, doing dumb things.

**Pull request merged**